spec/requests/api/v1/locations_spec.rb

```diff
@@ -21,4 +36,37 @@
     expect(actual_location["physical_address"]).to eq "Cr 70C No. 4-42 OF 203, Medellín"
     expect(actual_location["ethereum_address"]).to eq "0x7da07904ffc79d7a69afdecb7b1eb0d767b37803"
   end
-end
\ No newline at end of file
+
+  context "Sad paths" do
+    describe "If I forget a required field like 'creator' in the post form" do
+      it "I get a 400 for a bad request" do
+        post "/api/v1/locations", params: { 
+          location: { 
+            physical_address: "Cr 70C No. 4-42 OF 203, Medellín", 
+            city: "Medellín", 
+            state: "Medellín", 
+            country: "Colombia", 
+            ethereum_address: "0x7da07904ffc79d7a69afdecb7b1eb0d767b37803"
+          }
+        }
+
+        expect(response.status).to eq 400
+      end
+    end
+
+    describe "If I forget a required field like 'creator' in the post form" do
```

Comment:
@tmikeschu This has the same `description` as the test above.  Maybe it's something like: `When I provide an invalid/nonexistent ethereum_address`?  And the get request is _veeeeeery_ subtle.  How about using a variable like: `bad_ethereum_address = 0x7da0bad4ffc79d7a69afdecb7b1eb0d767b37803` and then for the get request, `get "/api/v1/locations/bad_ethereum_address"`.  
Thoughts?
Hopefully all this markup I'm adding is readable - apologies if it isn't!